tests/Jobs/KubernetesIngressDeleteJobTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Tests\Jobs;
+
+use App\User;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Tests\TestCase;
+use App\Jobs\KubernetesIngressDeleteJob;
+use App\Wiki;
+use App\WikiManager;
+use Illuminate\Contracts\Queue\Job;
+
+class KubernetesIngressDeleteJobTest extends TestCase
+{
+    use DatabaseTransactions;
+
+    public function testDoesNotDeleteNonDeletedWikiws()
+    {
+     
+        $user = User::factory()->create(['verified' => true]);
+        $wiki = Wiki::factory()->create( [ 'deleted_at' => null ] );
+        WikiManager::factory()->create(['wiki_id' => $wiki->id, 'user_id' => $user->id]);
+        
+        $mockJob = $this->createMock(Job::class);
```

Comment:
Is this how you are supposed to test these failure functions? It seems to work but this is the first time any of these failure things get tested afaik.

Reply:
Tried getting the `FailedJobProviderInterface`by using the app->make function which seemed to work, however in the tests nothing seems to get written to the database(also tried adding the Dispatchable interface to the job and use dispatchSync). Does that happen in production though?